snmp/tests/compose/data/_cisco-ipsec-flow-monitor.snmprec

```diff
@@ -0,0 +1,787 @@
+1.3.6.1.4.1.9.9.171.1.2.2.1.3.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|2|2
+1.3.6.1.4.1.9.9.171.1.2.2.1.4.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|4x|62757420717561696e746c79
+1.3.6.1.4.1.9.9.171.1.2.2.1.4.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|4x|627574207468656972207a6f6d62696573
+1.3.6.1.4.1.9.9.171.1.2.2.1.5.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|2|19
+1.3.6.1.4.1.9.9.171.1.2.2.1.5.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|2|24
+1.3.6.1.4.1.9.9.171.1.2.2.1.6.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|4|kept
+1.3.6.1.4.1.9.9.171.1.2.2.1.6.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|4|oxen
+1.3.6.1.4.1.9.9.171.1.2.2.1.7.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|4|oxen
+1.3.6.1.4.1.9.9.171.1.2.2.1.7.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|4|oxen
+1.3.6.1.4.1.9.9.171.1.2.2.1.8.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|2|5
+1.3.6.1.4.1.9.9.171.1.2.2.1.8.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|2|15
+1.3.6.1.4.1.9.9.171.1.2.2.1.9.1.52.100.114.105.118.105.110.103.32.102.111.114.119.97.114.100.32.111.120.101.110.32.111.120.101.110.32.116.104.101.105.114.32.102.111.114.119.97.114.100.32.111.120.101.110.32.102.111.114.119.97.114.100.2.12.98.117.116.32.113.117.97.105.110.116.108.121.19|2|24
+1.3.6.1.4.1.9.9.171.1.2.2.1.9.2.10.74.97.100.101.100.32.107.101.112.116.2.17.98.117.116.32.116.104.101.105.114.32.122.111.109.98.105.101.115.24|2|9
+1.3.6.1.4.1.9.9.171.1.2.3.1.1.7|2|7
+1.3.6.1.4.1.9.9.171.1.2.3.1.1.8|2|8
```

Comment:
```suggestion
```
Should we remove this from the snmprec file? As they are not accessible they probably should not be available

Same for `1.3.6.1.4.1.9.9.171.1.3.2.1.*`

Reply:
yes, also will remove the `1.3.6.1.4.1.9.9.171.1.3.2.1.1.*` !